.ghal.rules.json

```diff
@@ -320,6 +327,15 @@
           }
         }
       }
+    },
+    "labeled": {
+      "processor-conditional-release6": {
+        "type": "query",
+        "value": "length(Issue.labels[?name == ':checkered_flag: Release .NET 6']) != `0`",
```

Comment:
Should this be Issue.labels or something more like PullRequest.labels?

Also, we can probably remove the label sections for framework/winforms and framework/wpf, right? (I think those folders have been removed.)

Reply:
Thanks! Great copy/paste catch 😁